bundles/page/jackrabbit.rst

```diff
@@ -0,0 +1,61 @@
+Jackrabbit
+==========
+
+If you want to have the `versioning`_ feature or want to improve the performance you
+maybe come to the time to change from ``doctrinedbal`` phpcr implementation to ``jackrabbit``.
+
```

Comment:
Add installation requirements - at least `composer require jackalope/jackalope-jackrabbit`

Reply:
added in the installation part